src/components/ItemCard.js

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import CardSlide from './CardSlide';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+const ItemCard = () => {
+  const [roomData, setRoomData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/RoomData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => setRoomData(data));
+  }, []);
+
+  return (
+    <Card>
+      {roomData.map(
+        ({
+          host_id,
+          name,
+          address,
+          description,
+          bed,
+          price,
+          images,
+          review,
+        }) => {
+          const starRank = (Math.random() * 5).toFixed(2);
+          const reviewCount = review.length === undefined ? 'NEW' : starRank;
```

Comment:
mock 데이터에 review 구조를 보면 리뷰가 있는건 배열 안에 객체로 없는건 숫자로 되어있는데
실제 데이터는 review라는 키에 배열이 있고 리뷰가 있으면 객체 데이터로 채워지고, 없으면 그냥 빈 배열이 들어올 것 같습니다.

```suggestion
          const reviewCount = review.length ? 'NEW' : starRank;
```

그래서 목데이터에 review 부분에 리뷰가 없는것을 0 으로 하는게 아닌 빈배열로 두고
위와 같이 코드를 수정할 수 있겠습니다.